opendbc/car/interfaces.py

```diff
@@ -26,10 +26,6 @@
 ACCEL_MAX = 2.0
 ACCEL_MIN = -3.5
 
-# ISO 11270
-ISO_LATERAL_ACCEL = 3.0  # m/s^2
-ISO_LATERAL_JERK = 5.0  # m/s^3
-
```

Comment:
I had to move this if I want to be able to use them from `values.py` because otherwise I was getting a circular dependency